packages/core/src/interpreter.ts

```diff
@@ -282,13 +311,22 @@ export class Actor<
     }
     this.status = ActorStatus.Running;
 
+    const initEvent = createInitEvent(this.options.input);
+
+    this.system._sendInspectionEvent({
```

Comment:
I wish that this one would go through `system.sendTo` if that's treated as an explicit event here. It's not a blocker though since ultimately it doesn't matter that much and we can try to juggle things more in the future.